### PR TITLE
feat: sort by on quick filters and reset

### PR DESCRIPTION
**Pull request by egoroza**
Screen Shot 2022-10-13 at 11 57 00 AM
Screen Shot 2022-10-13 at 11 57 06 AM

### CODE REVIEW

**Comment by dyersituations** [comment]
Potential UI bug: looks like the filters don't quite line up, do we want these to be at the same level?
image

File: src/components/LoansByCategory/QuickFilters/QuickFilters.vue [hunk 1]
```diff
@@ -78,18 +98,27 @@ export default {
...
 	},
 	methods: {
 		updateLocation(location) {
 			this.updateFilters({ country: location });
+		},
+		resetFilters() {
+			this.$emit('reset-filters');
+			this.selectedGender = '';
+			this.sortBy = 'personalized';
+			this.updateLocation([]);
```
[the remark this answers is not in the document]
Comment (egoroza) [reply src/components/LoansByCategory/QuickFilters/QuickFilters.vue:121]: we have a separate ticket for tracking

**Review by dyersituations — APPROVED** [review]

**Review by roger-in-kiva — APPROVED** [review]
Looks good to continue, Im gonna fix ui bugs and some logic issues in other ticket.

**Review by mcstover — APPROVED** [review]
:)

**Pull request merged**